x-pack/plugins/maps/server/routes.ts

```diff
@@ -94,6 +94,6 @@ export async function initRoutes(core: CoreSetup, logger: Logger): Promise<void>
     }
   );
 
-  initMVTRoutes({ router, logger });
+  initMVTRoutes({ router, logger, core });
```

Comment:
Instead of passing in `coreSetup`, get coreStart from `core.getStartServices()` call on line 21